commands/create_cluster_test.go

```diff
@@ -131,13 +154,12 @@ func Test_CreateClusterSuccessfully(t *testing.T) {
 			verbose:             true,
 		},
 		addClusterArguments{
-			apiEndpoint:       mockServer.URL,
-			clusterName:       "Cluster Name from Args",
-			owner:             "acme",
-			token:             "fake token",
-			inputYAMLFile:     "testdata/minimal.yaml",
-			kubernetesVersion: "K8sVersionFromArgs",
-			verbose:           true,
+			apiEndpoint:   mockServer.URL,
+			clusterName:   "Cluster Name from Args",
+			owner:         "acme",
+			token:         "fake token",
+			inputYAMLFile: "testdata/minimal.yaml",
+			verbose:       true,
```

Comment:
No releaseVersion here?

Reply:
Good catch

Reply:
In fact, I didn't add it to all tests cases, since it's not a required argument.